### PR TITLE
Task/tap 2662

### PR DESCRIPTION
**Pull request by tamasveingartner**

### CODE REVIEW

File: InfluxWriter/2019.1.0/docs/Installation.md [hunk 1]
```diff
@@ -115,6 +115,10 @@ A sample configuration and an explanation of settings is given below.
     "BufferSize": 100, 
     "EscapeLineProtocol": false
   },
+  "Licensing": {
```
Comment (michaelgriffiths-mclaren) [inline InfluxWriter/2019.1.0/docs/Installation.md]: Have these options been added to the wrong version? The folder is 2019.1.0

File: InfluxWriter/2020.1/docs/Installation.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+## Influx Writer Configuration
+To configure the service, edit
+```
+/opt/MAT.TAP.AAS.InfluxDb.Writer.Hosting/appsettings.Production.json
+```
+
+**Please note that after you change settings in appsettings.Production.json in opt folder you need to restart service.**
+
+A sample configuration and an explanation of settings is given below.
+```json
```
Comment (michaelgriffiths-mclaren) [inline InfluxWriter/2020.1/docs/Installation.md]: This version is missing the license documentation

File: InfluxWriter/2019.1.0/docs/Installation.md (hunk 1 above)
Comment (tamasveingartner) [reply InfluxWriter/2019.1.0/docs/Installation.md]: yes, I reverted this commit.

File: InfluxWriter/2020.1/docs/Installation.md (hunk 2 above)
Comment (tamasveingartner) [reply InfluxWriter/2020.1/docs/Installation.md]: I have added the relevant changes.

**Pull request merged**